cmd/user/user.go

```diff
@@ -0,0 +1,214 @@
+// Licensed to Yugabyte, Inc. under one or more contributor license
+// agreements. See the NOTICE file distributed with this work for
+// additional information regarding copyright ownership. Yugabyte
+// licenses this file to you under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package user
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/yugabyte/ybm-cli/cmd/util"
+	ybmAuthClient "github.com/yugabyte/ybm-cli/internal/client"
+	"github.com/yugabyte/ybm-cli/internal/formatter"
+	ybmclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+)
+
+var UserCmd = &cobra.Command{
+	Use:   "user",
+	Short: "Manage users",
+	Long:  "Manage users in your YugabyteDB Managed account",
+	Run: func(cmd *cobra.Command, args []string) {
+		cmd.Help()
+	},
+}
+
+var listUsersCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List users",
+	Long:  `List users in your YugabyteDB Managed account`,
+	Run: func(cmd *cobra.Command, args []string) {
+		authApi, err := ybmAuthClient.NewAuthApiClient()
+		if err != nil {
+			logrus.Fatalf("could not initiate api client: %s", err.Error())
+		}
+		authApi.GetInfo("", "")
+
+		userListRequest := authApi.ListAccountUsers()
+
+		// if user filters by email, add it to the request
+		email, _ := cmd.Flags().GetString("email")
+		if email != "" {
+			userListRequest = userListRequest.Email(email)
+		}
+
+		resp, r, err := userListRequest.Execute()
+
+		if err != nil {
+			logrus.Debugf("Full HTTP response: %v", r)
+			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
+		}
+
+		userCtx := formatter.Context{
+			Output: os.Stdout,
+			Format: formatter.NewUserFormat(viper.GetString("output")),
+		}
+
+		if len(resp.GetData()) < 1 {
+			fmt.Println("No users found")
```

Comment:
don t use fmt ... if you need to exist fatal or error and exit is what you should use.

Reply:
logrus.Info should work I guess